core/src/main/java/org/kiwix/kiwixmobile/core/main/MainMenu.kt

```diff
@@ -133,6 +135,7 @@ class MainMenu(
     library.menuItemClickListener { menuClickListener.onLibraryMenuClicked() }
     readAloud.menuItemClickListener { menuClickListener.onReadAloudMenuClicked() }
     fullscreen.menuItemClickListener { menuClickListener.onFullscreenMenuClicked() }
+    newNavigation.menuItemClickListener { menuClickListener.onNewNavigationMenuClicked() }
```

Comment:
I also want to see this hidden if `!BuildConfig.Debug`, just to satisfy my paranoia